simpledotorg/simple-server
# Add diabetes missed visits graph

**Story card:** [sc-7782](https://app.shortcut.com/simpledotorg/story/7782/add-missed-visits-graph)

## Because

We need to add a graph showing the trend of diabetes missed visits patients on the diabetes reports page

## This addresses

- Region Summary and Region Summary Schema report blood sugar metrics
- Repository and Repository presenter present data to the controller
- The diabetes view adds html to display the diabetes missed visits graph

## Test instructions

1. Turn on `diabetes_management_reports` on flipper
2. Visit the reports page of a facility which has `enable_diabetes_management` set to true
3. Click on `Diabetes` link
4. You should see the Missed Visits graph
<img width="673" alt="Screenshot 2022-05-05 at 2 38 56 PM" src="https://user-images.githubusercontent.com/32141642/166893759-fc3d1962-2ae3-42ee-a87e-096d682a9eeb.png">
<img width="673" alt="Screenshot 2022-05-05 at 2 39 14 PM" src="https://user-images.githubusercontent.com/32141642/166893823-020fbe40-27e7-40c2-bb86-710f4cf95ac7.png">
